decidim-participatory_processes/app/models/decidim/participatory_process.rb

```diff
@@ -55,6 +58,12 @@ class ParticipatoryProcess < ApplicationRecord
     scope :upcoming, -> { where(arel_table[:start_date].gt(Time.current)) }
     scope :active, -> { where(arel_table[:start_date].lteq(Time.current).and(arel_table[:end_date].gt(Time.current).or(arel_table[:end_date].eq(nil)))) }
 
+    scope :private_spaces_user, lambda { |user|
```

Comment:
what you think if the scope name will be 'public_and_private_participatory_processes_for', as it returns all processes for the current_user?

Reply:
What about just returning the private ones instead and dealing with that at the `Queries` down below? In case not, what about `visible_participatory_processes_for`?